tests/run-duration.feature

```diff
@@ -0,0 +1,31 @@
+# This file is part of REANA.
+# Copyright (C) 2023 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
```

Comment:
This is a file meant for researchers, so adding a license there as we do for cluster component code is not really necessary.  (E.g. see we don't have it in `Snakefile` either.) Instead, we may perhaps advertise docs?

```
# Tests for workflow execution duration times
# http://docs.reana.io/....
```

(When we shall have a canonical location for user-facing Gherkin documentation, we could add the second line with a http to the thing. Currently, linking to https://continuous-reuse.docs.cern.ch/ may not be enough, and could be confusing. So, for now, just a one-line header comment explaining the purpose of the test file very briefly may be enough.)


P.S. I know that for the RootFit example we have already committed the test files with the full license header lines, but I think this would be better... since more user-facing. We may want to select something here and change the RooFit ones too accordingly.